buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -70,7 +76,15 @@ elif  [[ $(uname) == Darwin ]]; then
 fi
 
 # Install latest llvmlite build
-$CONDA_INSTALL -c numba llvmlite
+if [ $PYTHON \< "3.8" ]; then
+    $CONDA_INSTALL -c numba llvmlite
+else
+    $CONDA_INSTALL -c numba/label/dev_airflow llvmlite
```

Comment:
this should be reverted back to `-c numba` now

Reply:
fixed. hopes this is the last change